compiler/rustc_codegen_llvm/src/debuginfo/metadata.rs

```diff
@@ -875,7 +887,7 @@ pub fn build_compile_unit_di_node<'ll, 'tcx>(
             )
             // We get a path relative to the working directory from split_dwarf_path
             .map(|f| {
-                if tcx.sess.should_prefer_remapped_for_split_debuginfo_paths() {
+                if tcx.sess.should_prefer_remapped(RemapPathScopeComponents::DEBUGINFO) {
                     tcx.sess.source_map().path_mapping().map_prefix(f).0
                 } else {
                     f.into()
```

Comment:
This could be expressed as
```
tcx
 .sess
 .source_map()
 .path_mapping()
 .to_real_filename(f)
 .for_scope(RemapPathScopeComponents::DEBUGINFO)
 .to_string_lossy()
```